### ISSUE TITLE
MeshFaceMaterial generates errors until texture maps are loaded

### ISSUE THREAD

**Issue by makc** [mrdoob/three.js#8150]
So I have this
```javascript
	var urls = ['posx.jpg', 'negx.jpg', 'posy.jpg', 'negy.jpg', 'posz.jpg', 'negz.jpg'];
	var materials = [], textureLoader = new THREE.TextureLoader ();
	while (urls.length) {
		materials.push (new THREE.MeshBasicMaterial ({
			map: textureLoader.load (urls.shift ())
		}));
	}
	... new THREE.MeshFaceMaterial (materials) ...
```
and, before textures load, I get these:
![screen shot 2016-02-16 at 18 32 36](https://cloud.githubusercontent.com/assets/242577/13084946/18aa71ec-d4dc-11e5-92e6-e502256f05ec.png)

**Comment by makc** [comment]
And, changing the loop to
```javascript
	while (urls.length) {
		(function (material) {
			materials.push (material);
			var texture = textureLoader.load (urls.shift (), function () {
				material.map = texture; material.needsUpdate = true;
			});
		}) (new THREE.MeshBasicMaterial ({ overdraw: 0.5 }));
	}
```
makes them go away, so we know it's MeshFaceMaterial misbehaving not something else.

**Comment by mattdesl** [comment]
I think this might be fixed by my PR here: #8121

**Comment by makc** [comment]
so this is MeshBasicMaterial thing. I guess it's new to r74 because I do not remember seeing it before.
